### PR TITLE
Add info about media command for android

### PR DESCRIPTION
**Pull request by dshokouhi**
Docs for: https://github.com/home-assistant/android/pull/1513

### CODE REVIEW

File: docs/notifications/commands.md [hunk 1]
```diff
@@ -227,6 +228,37 @@ automation:
         title: "turn_off"
 ```
 
+## Media
+
+![Android](/assets/android.svg) &nbsp;<span class="beta">BETA</span><br />
+
+Users are able to control any active media session on their device. You must set `message: command_media` and the `title` must be one of the below list. The `channel` must be set to the package name you wish to control. The notification will post as normal if one of the required fields is left blank, has incorrect data or a media session is not active.
```
Comment (frenck) [inline docs/notifications/commands.md:235]:
```suggestion
Users are able to control any active media session on their devices. You must set `message: command_media` and the `title` must be one from the list below. The `channel` must be set to the package name you wish to control. The notification will post as normal if one of the required fields is left blank, has incorrect data or a media session is not active.
```

File: docs/notifications/commands.md [hunk 2]
```diff
@@ -227,6 +228,37 @@ automation:
...
+  - alias: Pause spotify
+    trigger:
+      ...
+    action:
+      service: notify.mobile_app_<your_device_id_here>
+      data:
+        message: "command_meda"
+        title: "pause"
+        data:
+          channel: "com.spotify.music"
```
Comment (frenck) [inline docs/notifications/commands.md:260]:
```suggestion
      - service: notify.mobile_app_<your_device_id_here>
        data:
          message: "command_meda"
          title: "pause"
          data:
            channel: "com.spotify.music"
```

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Comment (dshokouhi) [reply docs/notifications/commands.md:260]: Ok we will need to fix this across the board then 😅 as all of the examples are copied from one another.  Will fix this instance here but will submit a separate PR to update the rest as we have several areas to check.

**Review by frenck — APPROVED** [review]

File: docs/notifications/commands.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ The Companion apps offer a lot of different notification options. In place of po
 | `command_broadcast_intent` | Send a broadcast intent to another app, [see below](#broadcast-intent) for how it works and whats required. |
 | `command_dnd` | Control Do Not Disturb mode on the device, [see below](#do-not-disturb) for how it works and whats required. |
 | `command_high_accuracy_mode` | Control the high accuracy mode of the background location sensor, [see below](#high-accuracy-mode) for how it works and whats required. |
+| `command_media` | Control media playing on the device, [see below](#meda) for how it works and whats required. &nbsp;<span class="beta">BETA</span><br /> |
```
Comment (maxcanna) [inline docs/notifications/commands.md:26]: There's a typo, I think it should be `#media` 🙂

File: docs/notifications/commands.md [hunk 4]
```diff
@@ -227,6 +228,37 @@ automation:
...
+
+```yaml
+automation:
+  - alias: Pause spotify
+    trigger:
+      ...
+    action:
+      - service: notify.mobile_app_<your_device_id_here>
+        data:
+          message: "command_meda"
```
Comment (maxcanna) [inline docs/notifications/commands.md:257]: There's a typo, I think it should be #media 🙂

File: docs/notifications/commands.md (hunk 3 above)
Comment (dshokouhi) [reply docs/notifications/commands.md:26]: This is an old PR and I think someone fixed it earlier

File: docs/notifications/commands.md (hunk 4 above)
Comment (dshokouhi) [reply docs/notifications/commands.md:257]: This is an old PR and I think someone fixed it earlier

File: docs/notifications/commands.md (hunk 3 above)
Comment (maxcanna) [reply docs/notifications/commands.md:26]: Oh right, I spotted them in the changelog and didn't check on `master `

**Pull request merged**